scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
I noticed he looked different in that video you posted. Maybe I'll play around with model IDs a bit.

No one has commented on the non-retail Astral Flow setup too, I guess it's acceptable given it crashes otherwise? We had a Kirin Fenrir AF/BP one shot half our raid last night so I think having to deal with that every 10% on top of pets repopping is a good compromise.